app/models/hub_edos/student.rb

```diff
@@ -0,0 +1,32 @@
+module HubEdos
+  class Student
+    attr_accessor :terms_in_attendance, :academic_level
+
+    def initialize(uid)
+      @uid = uid
+      if uid.present?
+        @student_hash = HubEdos::V2::Student.new(user_id: uid).get
```

Comment:
I don't know if this is useful, but extracting student hash to a method would make this lazier (i.e. could initialize a list of these without triggering all the API calls until the data is requested)

Reply:
@developish - Yeah. I like that idea. Again, that could be a memoized request in it's own method.